includes/discovery/sensors/fanspeeds/f5.inc.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+if ($device['os'] == 'f5') {
+    $oids = snmp_walk($device, 'sysChassisFanSpeed', '-OsqnU', 'F5-BIGIP-SYSTEM-MIB');
+    d_echo($oids."\n");
+
+    if ($oids !== false) {
+        echo 'sysChassisFanSpeed ';
+    }
+
+    $divisor = 1;
+    $type    = 'f5';
+
+    foreach (explode("\n", $oids) as $data) {
+        $data = trim($data);
+        if ($data) {
+            list($oid,$descr) = explode(' ', $data, 2);
+            $split_oid        = explode('.', $oid);
+            $index            = $split_oid[(count($split_oid) - 1)];
+            $descr            = 'Fan Speed '.$index;
+            $oid              = '.1.3.6.1.4.1.3375.2.1.3.2.1.2.1.3.'.$index;
+            $fanspeed         = (snmp_get($device, $oid, '-Oqv') / $divisor);
```

Comment:
Not sure, I'm following?
I did the same code structure as in dell.inc.php in includes/discovery/sensors/fanspeeds. 
It also looks the same in the lmsensors.inc.php. They snmp_walk and then it snmp_get ?